src/prisma.ts

```diff
@@ -1,3 +1,11 @@
-import { PrismaClient } from '@prisma/client'
+import { Prisma, PrismaClient } from '@prisma/client'
 
-export const prisma = new PrismaClient()
+import { Environment } from './config.js'
+
+export const prisma = new PrismaClient({
+  ...(Environment.PRISMA_LOG
+    ? { log: ['query', 'info', 'warn', 'error'] }
+    : {}),
+})
+
+export { Prisma }
```

Comment:
I am fine with this but I prefer importing `Prisma` directly from `@prisma/client` as this tell us that `Prisma` is not custom type. What do you think?